lib/modules/manager/flux/types.ts

```diff
@@ -37,16 +37,31 @@ export interface HelmRepository extends KubernetesResource {
   };
 }
 
-export type FluxResource = HelmRelease | HelmRepository;
+export interface OCIRepository extends KubernetesResource {
```

Comment:
```suggestion
export interface OciRepository extends KubernetesResource {
```
would prefer to not have too many upper case letters.

Reply:
You are okay with having the TS interface name not match the Flux `OCIRepository` resource?

Reply:
Done.